### ISSUE TITLE
DATABASE: Simplify the screen for editing data sources

### ISSUE THREAD

**Issue by lbross** [USFS-PNW/Fia-Biosum-Manager#315]
and add the ability to select an SQLite datasource. Only keep the 'Get An MS Access Db File and Table' button. It is useful if a path to table gets messed up. Add a new button for the SQLite equivalent. These buttons will pop a screen that lets the analyst browse to a database and table, so using them is straightforward. Ultimately the MS Access button will go away. This is part of the transition to SQLite.

**Comment by lbross** [comment]
Sample of existing UI. As a first step for implementing this issue, let's agree on a new UI mock-up.

<img width="699" height="504" alt="Image" src="https://github.com/user-attachments/assets/d74a80dd-7c0f-41ea-a1ee-e59d37d3fbca" />

**Comment by jsfried** [comment]
Drue, this looks pretty streamlined; I am guessing that when you find the desired table, it and the database with path end up in the now empty fields in New Database File and Table? And then replace the Current ones after the Commit Change button is clicked? While the copy and move buttons does drop some functionality,  those functions don't seem that easy to document and ensure understanding of, and I can imagine users moving tables around by accident and breaking links in the project if they don't fully  understand copy and move operations.

**Comment by druepdx** [comment]
Yes, when the desired table is selected the New Database File And Table boxes are populated, and the actual overwriting happens with the Commit Changes button.
